x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/watcher/history/WatchRecord.java

```diff
@@ -43,12 +43,14 @@
     private static final ParseField METADATA = new ParseField("metadata");
     private static final ParseField EXECUTION_RESULT = new ParseField("result");
     private static final ParseField EXCEPTION = new ParseField("exception");
+    private static final ParseField EXECUTED_BY = new ParseField("executed_by");
```

Comment:
how about simply naming this `user`?

Reply:
one simple change to affect the whole PR ;)